### ISSUE TITLE
[ACCESSIBILITY] Maps - Filter/Location not receiving keyboard (SR) focus or being announced after applying filters

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#4420]
**Describe the issue**
The Maps filtering should work like the Events filtering in that the "Filter: Upcoming; Location: 1524" should be keyboard accessible and announced to Screen reader users upon landing back on page from applying filters. 

**To Reproduce**
**Steps to reproduce the behavior:**
1. Using Android, turn on TalkBack
2. Gesture to 'Map.'
3. Swipe through
4. Set a filter
5. Try to swipe through and ensure filter was applied

**Expected behavior**
"Filter: Upcoming; Location: 1524" can be gestured to using TalkBack so the user can ensure their filter parameters were applied. **(This should also be receiving the focus upon landing on Map page after applying filter parameters)**

**Actual behavior**
The "Filter: Upcoming; Location: 1524" is only read upon landing on the screen initially, this is not keyboard accessible to users gesturing through with TalkBack. Furthermore, the user lands on the back arrow receiving focus not "Filter: Upcoming; Location: 1524" 

**Screenshots**
![20241015_101837](https://github.com/user-attachments/assets/491328b5-21be-405d-a0b8-bc9b7d006a26)
This area is not keyboard accessible but should be and should receive focus after applying filters. 

https://github.com/user-attachments/assets/ed226bb3-5cf7-4922-b70b-c131d6c10ce4

**Testing Environment & Devices (please complete the following information):**
•	OS: Android Version 13
•	Device: Google Pixel 4a

**Additional context**
I know the receiving focus could technically be another bug since it was technically violating these:
[WCAG 2.4.7 Focus Visible (AA)](https://www.w3.org/TR/WCAG21/#focus-visible)
[WCAG 2.1.1 Keyboard](http://www.w3.org/TR/WCAG20/#keyboard-operation-keyboard-operable)

Please let me know if you'd like it to have a separate bug. I thought these go hand in hand perhaps though and was trying to not over inundate y'all. :)

**Comment by acovher2** [comment]
@pmarkhennessy @TodorBachvarov Due to the ADA title II changes and deadline of April 2026, I wanted to ensure this gets on the list and radar as we need to be compliant. This was on the PPT I sent you.

**Comment by acovher2** [comment]
https://github.com/user-attachments/assets/7d702cb7-e14d-49aa-8a60-a9164d579570

**Comment by acovher2** [comment]
https://github.com/user-attachments/assets/4b7d77fe-1b38-44dd-b177-7574ec8a7cbb
Retest in 7.3.5 failed. 

This does pronounce the Filter amount, but it is in the wrong order. This should gain focus immediately after applying new filters not at the end after many swipes.

**Comment by TodorBachvarov** [comment]
@acovher2 I've made some accessibility improvements for the new Maps UI in version 7.3.13. I think this is fixed. I've applied similar fix as the Event filters: Added the value of the filter and additional hint about the result into the button. Also the Focus is properly kept on the same button. As result when the filters are changed the result will be pronounced with the button (If the user don't like the value he can easily change it again by just double tapping on the button as it's focussed)
